modules/installation-configuration-parameters.adoc

```diff
@@ -707,6 +707,23 @@ ifdef::agent[]
 Supported values are `3`, `4`, `5`, or `1` when deploying {sno}.
 endif::agent[]
 
+a|
+----
+arbiter:
+    name: arbiter
+----
+|The {product-title} cluster requires a name for arbiter nodes.
+|String. For example, `arbiter`.
+
+
+a|
+----
+arbiter:
+    replicas: 1
+----
+|The `replicas` parameter sets the number of arbiter nodes for the {product-title} cluster.
+|`1`
+
```

Comment:
these new rows of the table are not displaying properly and have bumped the entire table so much that is unreadable; this is a merge-blocker
i've given suggestions here for what I _think_ will fix it, but you might need to fiddle with it a bit to get the columns back to regular size and the content in the correct ones